src/explorer/decorationProvider.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { Disposable } from "vscode";
+
+class DecorationProvider implements vscode.FileDecorationProvider {
+    private disposables: Disposable[];
+    private decoration: vscode.FileDecoration = new vscode.FileDecoration("", "has conflicts", new vscode.ThemeColor("inputValidation.warningBorder"));
```

Comment:
use undefined instead of empty string.  Don't use empty string unless you have a clear reason.

Reply:
`inputValidation.warningBorder` doesn't make sense to me. it's not in a input box, and why use border color? 

> list.warningForeground: Foreground color of list items containing warnings.

maybe this one?

Reply:
Thanks! I just search for `warning` and choose the first color.